src/components/stockItemLists.vue

```diff
@@ -35,14 +40,18 @@
     <button @click="getByClient" id="getByClient">Get Data</button>
   </div>
   <br/>
-  <div v-if="clientId">
-  <label for="clientIdDisplay" id="clientIdDisplay">Client ID</label>
-  {{ this.clientId }}
-  </div>
 
   <div>
-    <table v-if="produceByClient.length > 0">
+    <table id="pdfMaker2" v-if="produceByClient.length > 0">
```

Comment:
Could have more meaningful names in the future instead of pdfMaker, pdfMaker2, etc.

Reply:
They were added as id for selected data when the pdf document is produced. They can be updated to have more meaningful names such as one describing content of table.